.gitpod.yml

```diff
@@ -0,0 +1,4 @@
+image:
+  file: .gitpod.Dockerfile
+tasks:
+  - init: pip install -r requirements-dev.txt && make develop
```

Comment:
Can you use conda here instead of pip?

Reply:
Sure, any reason to prefer conda over pip? Is that the preferred development environment for pandas?